packages/arb-token-bridge-ui/src/util/withdrawals/fetchWithdrawalsFromSubgraph.ts

```diff
@@ -14,6 +14,10 @@ export type FetchWithdrawalsFromSubgraphResult = {
   l2BlockTimestamp: string
   l2TxHash: string
   l2BlockNum: string
+  direction: 'deposit' | 'withdrawal'
```

Comment:
same as the other, i expect it to be `withdrawal` only

Reply:
Updated